### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by oonasaksman**
## Describe your changes

Implemented navbar component. Simple code so no comments.

## Checklist before requesting a review

-   [ x] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added thorough tests

### CODE REVIEW

**Review by Marakke — CHANGES_REQUESTED** [review]
The Header component should probably be removed now that the NavBar is implemented.

File: frontend/src/components/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import PersonIcon from '@mui/icons-material/Person';
+import KnowItLogo from './knowit.svg';
+import AVALogo from './ava.svg';
+
+const NavBar = () => {
+    return (
+        <AppBar
+            className="navigation"
+            elevation={0}
+            style={{ backgroundColor: '#333333', position: 'relative' }}
```
Comment (Marakke) [inline frontend/src/components/NavBar.tsx:12]: This styling should be moved to the SCSS file.

File: frontend/src/components/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                className="logo"
+                src={AVALogo}
+                alt="Anonymous voting app logo"
+            />
+
+            <Typography className="login">Login</Typography>
+
+            <PersonIcon
+                className="profile"
+                style={{ fontSize: 40, color: '#333333' }}
```
Comment (Marakke) [inline frontend/src/components/NavBar.tsx:26]: Same here, move this to SCSS.

**Review by Marakke — CHANGES_REQUESTED** [review]

File: frontend/src/components/NavBar.tsx (hunk 1 above)
Comment (Marakke) [reply frontend/src/components/NavBar.tsx:12]: Also, you should `@import '../styles/_colors';` and use the color variables defined there. No need to define them in each file.

File: frontend/src/components/App.tsx [hunk 3]
```diff
@@ -29,6 +30,7 @@ function App() {
 
     return (
         <div className="App">
+            <NavBar></NavBar>
             <Header></Header>
```
Comment (vrindavj) [inline frontend/src/components/App.tsx:34]: Header component can be removed as we have navabar

**Review by vrindavj — APPROVED** [review]
Rest everything looks good to me.:)

**Review by Marakke — DISMISSED** [review]

**Review by Marakke — DISMISSED** [review]

**Review by Marakke — DISMISSED** [review]

**Review by vrindavj — APPROVED** [review]
header component is removed and code is working fine👍🏻

**Comment by Jontzii** [comment]
Fixed rebase issue

**Review by Marakke — APPROVED** [review]

**Review by JeToKo — APPROVED** [review]
Looks good

**Pull request merged**